### PR TITLE
Modding updates

### LINKED ISSUE #1263: Outline/visibility for BN/QAT presence in beatmap discussions
Users should be able to identify the BN and QAT feedback at a glance similar to how the current forum highlights their posts with a golden header.

Perhaps consider a lesser form of highlighting for highly upvoted feedback as well.

### LINKED ISSUE #1267: Show warning notification when posting 'duplicate' feedback in beatmap discussions
When posting a comment with the same timestamp as existing feedback (either resolved or unresolved), the page should direct you to the existing feedback first, prompting the user to check if their concern is already being addressed or has been addressed.

This will prevent mass feedback duplication on particularly problem maps and is also the logical expectation for timestamp-contingent posting.

### PR DESCRIPTION
**Pull request by nanaya**
Fixes #1267 (kind of), fixes #1263.

Timestamp check (temp). Can use better wording.

New timestamp link style and moved username prefixed to the message

Indicator for admin/qat/bng.

And map owner/mapper. Not sure what the badge should say.

### CODE REVIEW

**Comment by Ephemeralis** [comment]
Hot stuff. Big visual improvement on the prefixed username + new timestamp.

File: resources/assets/coffee/_classes/beatmap-discussion-helper.coffee [hunk 1]
```diff
@@ -61,3 +74,14 @@ class @BeatmapDiscussionHelper
...
+
+
+  @moderationGroup: (user) =>
+    if user.groups?
+      _.intersection(user.groups, ['admin', 'qat', 'bng'])[0]
+    else
+      switch
+        when user.isAdmin then 'admin'
+        when user.isQAT then 'qat'
+        when user.isBNG then 'bng'
```
Comment (peppy) [inline resources/assets/coffee/_classes/beatmap-discussion-helper.coffee]: how does this look if changed to `nominator`?

Comment (nanaya) [inline resources/assets/coffee/_classes/beatmap-discussion-helper.coffee]: I think it'll be a bit too long. "mapper" is already overflown and anything longer would require a bit more adjustments.

Comment (peppy) [inline resources/assets/coffee/_classes/beatmap-discussion-helper.coffee]: the other option is to display "BN" but give these tooltips

qat -> Quality Assurance Team
bn -> Beatmap Nominator

Comment (nanaya) [inline resources/assets/coffee/_classes/beatmap-discussion-helper.coffee]: that could work.

**Comment by peppy** [comment]
going to merge this for the time being, but the BNG will need to change pending design amendments.

**Pull request merged**